### ISSUE TITLE
[Twitch] Unable to play subscriber-only VODs

### ISSUE THREAD

**Issue by zimmerry** [futo-org/grayjay-android#482]
Tested on multiple VODs from Critical Role. Attempting to play a video fails with the error

`[Twitch (Beta)] This content is restricted to subscribers (748)[8-9]`

I am subscribed to the channel and I'm able to view these videos on the web and on Twitch's official app.

![Screenshot_20231210-134235.png](https://github.com/futo-org/grayjay-android/assets/53264075/c9353184-3fa2-4c9f-83fe-c5abd50cc0a5)

![Screenshot_20231210-134228.png](https://github.com/futo-org/grayjay-android/assets/53264075/7e594719-402b-465d-91c9-0aa830819a7e)

![Screenshot_20231210-134223.png](https://github.com/futo-org/grayjay-android/assets/53264075/a7b7c706-1a87-45dd-bde2-3d80acb5bc88)

**Comment by Kelvin-FUTO** [comment]
Makes sense, we don't use your login to obtain video details. Should be a fairly easy addition if Twitch doesn't do anything special. We can make this a Twitch plugin setting.

**Comment by fabio-87** [comment]
This still happens to me too. I'm a subscriber, but it's impossible to view videos. There are any news about it?

**Comment by kaidelorenzo** [comment]
If this is still not working for some please re-open the issue
